### DISCUSSION TITLE
Implimentation of GTransformer from GROVER

### DISCUSSION THREAD

**Discussion by gunabam** [pyg-team/pytorch_geometric#7266] [Ideas]
There is a GTransformer architecture from the paper "Self-Supervised Graph Transformer on Large-Scale Molecular Data". The github link is https://github.com/tencent-ailab/grover. Has this been implemented with the pyG api? If not, can this be considered for future implementations, as this model preforms better than AttentiveFP (existing molecular implementation with pyG).

**Comment by rusty1s** [comment]
Thanks for the pointer. This is very interesting and a cool contribution to PyG. We currently do not have plans to implement this on our own, so I label this as "Help wanted" if anyone is interested to contribute this.

**Comment by FarzanT** [comment]
Hi, a more recent method from the paper [Geometry-enhanced molecular representation learning for property prediction](https://www.nature.com/articles/s42256-021-00438-4) seems to outperform GROVER. They also have their own official implementation [here](https://github.com/PaddlePaddle/PaddleHelix/tree/dev/apps/pretrained_compound/ChemRL/GEM).  Just in case someone wanted to prioritize the SOTA.

Edit: I've realized that there's an update to GEM, [GEM-2](https://arxiv.org/abs/2208.05863), and neither of these are SOTA on some of the benchmarks they were superior on before (e.g. the [OGB Quantum Chemistry Leaderboard](https://ogb.stanford.edu/docs/lsc/leaderboards/#pcqm4mv2)). [Uni-Mol+](https://github.com/dptech-corp/Uni-Mol/tree/main/unimol_plus) seems to be the the top method at the moment.
